changelog/1718.breaking.rst

```diff
@@ -0,0 +1,3 @@
+The parameters ``Z`` and ``mass_numb`` are now |keyword-only| for
+`~plasmapy.particles.symbols.ionic_symbol` and
+`~plasmapy.particles.symbols.particle_symbol`.
```

Comment:
A lot of breaking changes in these changelogs, recently. I'm actually kind of happy to see it, but from a semver standpoint, it's... 😅

Reply:
Agreed!